.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: c
```

Comment:
Why anything at all. All that `language: c` does it provides a build environement for C and some default test scripts.